python/pyrogue/_Block.py

```diff
@@ -15,6 +15,14 @@
 import pyrogue as pr
 
 
+def startTransaction(block, *, type, forceWr=False, checkEach=False, variable=None, index=-1, **kwargs):
```

Comment:
Should there be a print statement about how `startTransaction` is depreciated and should use `_startTransaction` instead?

Reply:
Designs will need to use the startTransaction and checkTransaction helper functions in the future. I can add a note to the top of block.py.